wiki/jee.md

```diff
@@ -361,6 +363,102 @@ In order to do this, you must provide the following content in the
 
 ... here, the `<class-loader delegate="false" />` part is the relevant bit.
 
+### Deactivate MoxyJson
```

Comment:
Are there any articles / bug reports you can link to for this issue?

In todays daily I learned that we may require some kind of agreement regarding release of copyright or some such from you in order to be able to include this. However, David is the one that knows all about this and he's got the day off ... I'll be in touch ASAP with details :)